run.sh

```diff
@@ -152,7 +152,11 @@ if [ "$DOWNLOAD_GENESIS" == "1" ]; then
 fi
 
 # Validate genesis
-[ "$VALIDATE_GENESIS" == "1" ] && $PROJECT_BIN validate-genesis
+# NOTE: e-money currently doesn't have a `validate-genesis` command.
+if [ $PROJECT_BIN != "emd" ]; then
+  echo "Validating"
+  [ "$VALIDATE_GENESIS" == "1" ] && $PROJECT_BIN validate-genesis
+fi
```

Comment:
It looks like you can just set this as an environment variable.  See [here](https://github.com/ovrclk/cosmos-omnibus/blob/a73e67fd8c03ec3c3e3f642c5d6b7a0b3f6ae39f/sifchain/deploy.yml#L13)

Reply:
Thanks for the help, reverted and added env